### DISCUSSION TITLE
Icon request: icon-wheelbarrow

### DISCUSSION THREAD

**Discussion by sammead92** [FortAwesome/Font-Awesome#21226] [Icon Requests]
Wheelbarrow icon needed for website I am building.

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Please fix this request according to our [Contributing guidelines](https://github.com/FortAwesome/Font-Awesome/blob/master/CONTRIBUTING.md#requesting-new-icons).

**Comment by YlloNieR** [comment]
+1

**Comment by YlloNieR** [comment]
+1

**Comment by slekrem** [comment]
+1

**Comment by willidh** [comment]
+1

**Comment by websolplus** [comment]
+1

**Comment by JoshWeb** [comment]
+1

**Comment by hadees** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
